contract/src/orca.flows.js

```diff
@@ -93,24 +88,29 @@ export const makeCreateAndFund = async (
 
   // vstorage tests
   trace('writing');
-  // setValue(`status x`)
-  // const node1 = await makeChildNode(`orca-createAndFund-${localAddress.value}-${localAddress.value}`);
 
   trace('localAddress', localAddress);
   trace('remoteAddress', remoteAddress);
-  trace('fund new orch account');
+  trace('fund new orch account 2');
   trace('seat', seat);
-  trace('transfer', transfer);
-  await transfer(
-    seat,
-    localAccount,
-    remoteAccount,
-    give,
-    amt,
-    localAddress,
+
+  await localTransfer(seat, localAccount, give);
+  trace('after transfer');
+
+  // const localChainBalance = await localAccount.getBalance("BLD"); //'ubld'
+  // trace("localChainBalance", localChainBalance);
+
+  await localAccount.transfer(
+    {
+      denom: 'ubld',
+      value: amt.value / 2n,
+    },
     remoteAddress,
   );
   seat.exit();
+  const remoteChainBalance = await remoteAccount.getBalance('uosmo');
+  console.log('remoteChainBalance', remoteChainBalance);
+
```

Comment:
The console log has limited visibility. Maybe put the balance in vstorage so the client can get it?